src/Packetery/Module/Checkout.php

```diff
@@ -1038,4 +1026,12 @@ private function getTaxClassWithMaxRate() {
 		return $resultTaxClass;
 	}
 
+	/**
+	 * Tells cart price with VAT and without Discounts.
```

Comment:
V minulosti jsem navrhl psát "without discounts", to byla chyba. Ve skutečnosti to není "beze slev" ale "se započítanými slevami" takže bych buď kopíroval konvenci WC - getCartContentsTotalIncludingTax nebo použil anglicky správné "including discounts".